.github/workflows/flake8.yml

```diff
@@ -0,0 +1,22 @@
+name: Flake8
+
+on: [push, pull_request]
+
+jobs:
+  flake8-lint:
+    runs-on: ubuntu-latest
+    name: Lint
+    steps:
+      - name: Check out source repository
+        uses: actions/checkout@v3
+      - name: Set up Python environment
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
+      - name: flake8 Lint
+        uses: py-actions/flake8@v2
+        with:
+          ignore: "E203,E266,E501,W503,F403,F401"
```

Comment:
is there any way to use the config from the file `.flake8`?

Reply:
maybe just having a single job that runs the command `pre-commit run --all-files`?
just asking, I don't have any strong opinion on this 😅